rest-openehr/pom.xml

```diff
@@ -35,6 +35,8 @@
 
     <properties>
         <license-header.dir>../</license-header.dir>
+        <ipf-atna-spring-boot-starter.version>4.0.0</ipf-atna-spring-boot-starter.version>
+        <java-jwt.version>3.3.0</java-jwt.version>
```

Comment:
The version should be managed in the server pom, or maybe in the bom;
see duplication in `service`